src/Ztm.WebApi/Watchers/TransactionConfirmation/EntityRuleRepository.cs

```diff
@@ -0,0 +1,247 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using NBitcoin;
+using Newtonsoft.Json;
+using Ztm.Data.Entity.Contexts;
+using TransactionConfirmationWatchingRuleModel = Ztm.Data.Entity.Contexts.Main.TransactionConfirmationWatchingRule;
```

Comment:
I think we can make it shorter by rename to `EntityModel`.

Reply:
Good idea